myhpi/static/scss/myHPI.scss

```diff
@@ -5,20 +5,25 @@
 $primary: #F5631E;

```

Comment:
Making the breadcrumbs gray again could help differentiating them from the page content:

![grafik](https://github.com/user-attachments/assets/3929e437-9fa9-4f5d-88fe-7a464689369b)


Before:
![grafik](https://github.com/user-attachments/assets/ffeececf-77da-4d21-b75b-96da958eae85)